src/Forms/InputLabel.tsx

```diff
@@ -56,6 +56,17 @@ export interface InputLabelProps {
   hideError?: boolean;
 }
 
+const accessibility = `
```

Comment:
I think this needs a more descriptive name, maybe something like `hideFromDisplay`? Because while this is an accessibility feature it's really only dealing with one part of accessibility, so adding it doesn't necessarily make a component accessible, and the removing it doesn't make it inaccessible.

Reply:
One other thought: Maybe we should move this into its own file (e.g. `src/utils/mixins.ts` or `src/utils/accessibility.ts`) so that we can import this into other components where need to hide text from the UI. Along the same lines as Natalyn's work in #80, this would make it easier for us to reuse common solutions across the library.

Reply:
I like the idea of having this in a new component but can we open a new ticket for this because my course planning ticket is depend on this functionality.
We can split this to a new component later if that make sense to you.

Reply:
Yep, that's a good idea to wait on splitting this out.